### PR TITLE
Zepto API doc guides

### PR DESCRIPTION
**Pull request by rasika-zepto**
* Setup Zepto API guides on readme.com
* Setup Zepto API Changelogs on readme.com

### CODE REVIEW

File: source/changelogs/2020-12-15.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: "2020-12-15"
+excerpt:
+category: 65ada3d7699b7900765d2ed5
+hidden: false
+type: added
+---
+
+* Add changelog
+* Re-word Payment Requests introduction to better cover its use with Receivable Contacts.
```
Comment (zubin) [inline source/changelogs/2020-12-15.md]: "Reword" doesn't need to be hyphenated. 

```suggestion
* Reword Payment Requests introduction to better cover its use with Receivable Contacts.
```

File: source/changelogs/2020-12-16.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: "2020-12-16"
+excerpt:
+category: 65ada3d7699b7900765d2ed5
```
Comment (zubin) [inline source/changelogs/2020-12-16.md]: What does this refer to?

File: .github/workflows/rdme-changelogs.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+name: Sync ReadMe Changelogs 🦉
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  rdme-changelogs:
```
Comment (zubin) [inline .github/workflows/rdme-changelogs.yml]: Would `readme-changelogs` be more readable? (ditto for filename)

**Review by zubin — APPROVED** [review]
Some non-blocking suggestions. LGTM 👍

File: source/changelogs/2020-12-16.md (hunk 2 above)
Comment (rasika-zepto) [reply source/changelogs/2020-12-16.md]: Readme.com uses [yaml front matter](https://jekyllrb.com/docs/front-matter/) to push metadata along with the markdown files. The `category` identifies which section of docs where the markdown should go into. It's not required in the case of changelogs, so I'll remove these 👍

File: .github/workflows/rdme-changelogs.yml (hunk 3 above)
Comment (rasika-zepto) [reply .github/workflows/rdme-changelogs.yml]: `rdme` is the name of the [official cli tool](https://docs.readme.com/main/docs/rdme) for readme.com, and the naming was automatically generated by the tool. I can change this 👍

**Pull request merged**